app/results.py

```diff
@@ -188,8 +188,12 @@ def get(self):
             # If a query looks like a phone number, show the user a result
             # of looking up the number in the carriers-provided BBS system.
             if self.config.jp_mobile_carrier_redirect:
-                if jp_mobile_carriers.handle_phone_number(self, query.query):
-                    return 
+                try:
+                    if jp_mobile_carriers.handle_phone_number(self, query.query):
+                        return
+                except Exception as e:
+                    logging.error('failed to scrape phone number')
```

Comment:
Thanks for the pointer! I have further questions:
- Do we need a message with i18 translation support in this case as we show the message to users? (I guess that _("a message") means in the example). If not, then I'll just update with some clearer message (e.g., "failed to scrape the search result for the phone number").
- Should the return code be 500 (internal server error) in this case?

(Sorry for asking too many little details. I'm new to the codebase so please bear with me for now.)